doc/api/process.md

```diff
@@ -1893,8 +1893,16 @@ A number which will be the process exit code, when the process either
 exits gracefully, or is exited via [`process.exit()`][] without specifying
 a code.
 
-Specifying a code to [`process.exit(code)`][`process.exit()`] will override any
-previous setting of `process.exitCode`.
+The value of `process.exitCode` can be updated by either setting the value
+directly (e.g. `process.exitCode = 9`), or by specifying a code to
+[`process.exit(code)`][`process.exit()`] (the latter takes
+precedence and will override any previous setting of `process.exitCode`).
```

Comment:
I wonder if it wouldn't be better to give examples instead


````suggestion
The value of `process.exitCode` can be updated by either assigning a value to
`process.exitCode` or by passing an argument to [`process.exit()`][]:

```console
$ node -e 'process.exitCode = 9'; echo $?
9
$ node -e 'process.exit(42)'; echo $?
42
$ node -e 'process.exitCode = 9; process.exit(42)'; echo $?
42
```
````

Reply:
Sounds good to me 🙂 :+1: